### DISCUSSION TITLE
The "Go to Desktop" button in the lower right corner of win 10

### DISCUSSION THREAD

**Discussion by WoodPig4869** [valinet/ExplorerPatcher#3331] [Ideas]
I think this option should add the function of adjusting button size
that would be awesome

**Comment by pyrates999** [comment]
Install 7+ Taskbar Tweaker.  Right click on it's icon in the system tray and select advanced options.  Then adjust the option show_desktop_button_size and set that to 12 or 16 if you want it wider.
